_tests/integration/docker_create_bitrise.yml

```diff
@@ -0,0 +1,45 @@
+format_version: 1.3.0
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+workflows:
+  docker-create-fails-invalid-port:
+    title: Expected to fail on docker create, invalid port provided
```

Comment:
nit: the `title` field is truncated to 80 or so chars at runtime, but we also have a `description` field for longer text